File: benchmarks/competitors/array.c
#include <stdio.h>

int main(void) {
    long long values[5] = {1, 2, 3, 4, 5};
    long long total = 0;
    for (long long i = 0; i < 5; i++) {
        total += values[i];
    }
    printf("%lld\n", total);
    return 0;
}

File: benchmarks/competitors/array.rs
fn main() {
    let values: [i64; 5] = [1, 2, 3, 4, 5];
    let mut total: i64 = 0;
    let mut i = 0usize;
    while i < values.len() {
        total += values[i];
        i += 1;
    }
    println!("{total}");
}
